packages/jest-snapshot/src/Snapshot.js

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright (c) 2014-present, Facebook, Inc. All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+'use strict';
+
+const fs = require('fs');
+
+class Snapshot {
+
+  constructor(filename) {
+    this._filename = filename;
+    this._loaded = null;
+  }
+
+  _getLoaded() {
```

Comment:
I do this because I save after each file, so the next time it run the test needs to be refreshed, I want to refactor this by patching Jasmine to give me the information of whenever a Test/Suite is done, so I can store the whole object at once, this was just easier for starting